autocomplete/.vscode/tasks.json

```diff
@@ -45,6 +55,7 @@
 				"isDefault": true
 			},
 			"dependsOn": [
+				"Build EmmyLua Aliases",
```

Comment:
Since all of the autocomplete is technically EmmyLua, I don't think this needs specifying. Perhaps this could also just be put into the regular builder instead of a separate step. I have never looked at the builder though and might not be the best to give my input on this.